images/hubble-proto/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM docker.io/library/golang:1.15-alpine3.12 as builder
```

Comment:
@tklauser will this be automatically updated by the script used to update golang version?

Reply:
Yes, it will. But in it's current version it will strip  `-alpine3.12`. Will update the `sed` command in the upcoming Go 1.15.1 bump to retain it.